src/functions-reference/integer-valued_basic_functions.Rmd

```diff
@@ -135,12 +135,13 @@ This is a no-op. \[ \text{operator+}(x) = x \]
 
 ## Absolute functions
 
-<!-- R; abs; (T x); -->
-\index{{\tt \bfseries abs }!{\tt (T x): R}|hyperpage}
+<!-- T; abs; (T x); -->
+\index{{\tt \bfseries abs }!{\tt (T x): T}|hyperpage}
 
-`R` **`abs`**`(T x)`<br>\newline
-absolute value of x
-`r since("2.0")`
+`T` **`abs`**`(T x)`<br>\newline
+The absolute value of x. This function works elementwise over containers such as
```

Comment:
I'd like to see `T` described as applying to any real or integer value or real or integer-valued container.    As is, it never says it can apply to `real` or `int`.